bitbots_quintic_walk/include/bitbots_quintic_walk/QuinticWalkingNode.hpp

```diff
@@ -45,155 +45,175 @@ The original files can be found at:
 #include <unistd.h>
 
 
-class QuinticWalkingNode {
-public:
-    QuinticWalkingNode();
+namespace bitbots_quintic_walk {
 
-    /**
-     * This is the main loop which takes care of stopping and starting of the walking.
-     * A small state machine is tracking in which state the walking is and builds the trajectories accordingly.
-     */
-    void run();
+    struct MessageHandlers {
+        tf::TransformBroadcaster odom_broadcaster;
 
-    /**
-     * Dynamic reconfigure callback. Takes in new parameters and applies them to the needed software parts
-     */
-    void reconf_callback(bitbots_quintic_walk::bitbots_quintic_walk_paramsConfig &config, uint32_t level);
+        ros::Publisher pubControllerCommand;
+        ros::Publisher pubOdometry;
+        ros::Publisher pubSupport;
+        ros::Publisher pubDebug;
+        ros::Publisher pubDebugMarker;
 
-    /**
-     * Initialize internal WalkEngine to correctly zeroed, usable state
-     */
-    void initializeEngine();
+        ros::Subscriber subCmdVel;
+        ros::Subscriber subRobState;
+        ros::Subscriber subJointStates;
+        ros::Subscriber subKick;
+        ros::Subscriber subImu;
+        ros::Subscriber subCopL;
+        ros::Subscriber subCopR;
+    };
```

Comment:
why are you using a struct? What's the advantage?

Reply:
That's a good question. There is no definitive reason. I just _felt_ it was somehow better have fewer direct instance variables but now I do agree that it is sort of useless

Reply:
I removed the struct